include/room_messages.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+//
+
+#pragma once
+
+#include "MessageIdentifiers.h"
+
+enum RoomMessages
+{
+    ID_ROOM_CHAT = ID_USER_PACKET_ENUM + 1,
```

Comment:
Preferably lets use identifiers like Chat, Data, Beacon, etc (In UpperCamelCase)